docs/source/install.rst

```diff
@@ -0,0 +1,129 @@
+Installation
+============
+
+Clone the repository:
+
+.. code-block:: bash
+
+    $ git clone git@github.com:bigchaindb/bigchaindb-examples.git
+
+Go into it!
+
+.. code-block:: bash
+
+    $ cd bigchaindb-examples
+
+We now document two options:
+
+* One that is considered easy, if you consider using docker to be easy.
+* One that is considered to be hard, if you consider installing different
+  components, and required dependencies to be rarely a smooth process.
+    
+
+The Easy Way
```

Comment:
It'd be nice to suffix all the easies with (Docker) and the hards with (Manual) just to make it clear.